e2e/migrations/migrations_test.go

```diff
@@ -108,15 +109,26 @@ func isSuccess(execCmd *exec.Cmd, retries int, keyword string) (string, error) {
 
 // allNodesAreReady attempts to query the status of a cluster a specific number
 // of times
-func allNodesAreReady(retries int) (string, error) {
-	cmd := exec.Command("nomad", "node-status")
+func allNodesAreReady(retries int, flags string) (string, error) {
```

Comment:
I'm not a huge fan of these. They seem super fragile but I couldn't see a better way with the commands we have. This is a sign that we need to improve them

Reply:
I completely agree. Alternatively, I could do these checks via the HTTP API, which would be less fragile but would not be a complete e2e test. Open to any other suggestions.